CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 **This is the changelog for the core Rust library**. There's a [separate changelog](./python/CHANGELOG.md) for the Python bindings.
 
+## [0.4.0] - <RELEASE_DATE>
```

Comment:
Are either of these breaking changes? Can it be 0.3.1?

Reply:
None of these changes are breaking. I've changed the version number to 0.3.1.